torchensemble/_base.py

```diff
@@ -349,8 +357,8 @@ def __init__(self, input_dim, output_dim, depth=5, lamda=1e-3, cuda=False):
 
         self._validate_parameters()
 
-        self.internal_node_num_ = 2 ** self.depth - 1
-        self.leaf_node_num_ = 2 ** self.depth
+        self.internal_node_num_ = 2**self.depth - 1
```

Comment:
Several lines here do not meet the coding style requirements. Please consider to execute the command `black --skip-string-normalization --config pyproject.toml torchensemble/` in the root directory of the project, which will automatically format the code

Reply:
Weird, I ran the same `black` command and it automatically did the changes mentioned above. I reverted them manually now. Maybe it has something to do with my version of `black`.